plugins/modules/rds_subnet_group.py

```diff
@@ -65,60 +66,50 @@
     returned: I(state=present)
     type: complex
     contains:
-        name:
+        db_subnet_group_name:
```

Comment:
I was wondering about that. Would it be acceptable to do both (return the new keys AND the old ones for backward compatibility)?

Reply:
Perfectly acceptable, and personally I prefer it that way.  We've a mix of modules originally written of boto v2 and some in boto v3, in the long term it's more consistent for us to return the boto3 style values everywhere, but we need to avoid breaking things for folks used to the 'old' return codes from the module.